library/frr_bgp.py

```diff
@@ -0,0 +1,410 @@
+                type: bool
+              update_source:
+                description:
+                  - Source of the routing updates.
+              password:
+                description:
+                  - Password to authenticate the BGP peer connection.
+              enabled:
+                description:
+                  - Administratively shutdown or enable a neighbor.
+                type: bool
+              description:
+                description:
+                  - Neighbor specific description.
+              ebgp_multihop:
+                description:
+                  - Specifies the maximum hop count for EBGP neighbors not on directly connected networks.
+                  - The range is from 1 to 255.
+                type: int
+              peer_group:
+                description:
+                  - Name of the peer group that the neighbor is a member of.
+              timers:
+                description:
+                  - Specifies BGP neighbor timer related configurations.
+                suboptions:
+                  keepalive:
+                    description:
+                      - Frequency (in seconds) with which the FRR sends keepalive messages to its peer.
+                      - The range is from 0 to 65535.
+                    type: int
+                    required: True
+                  holdtime:
+                    description:
+                      - Interval (in seconds) after not receiving a keepalive message that FRR declares a peer dead.
+                      - The range is from 0 to 65535.
+                    type: int
+                    required: True
+              activate:
+                description:
+                  - Enable the address family for this neighbor.
+                type: bool
+              remove_private_as:
+                description:
+                  - Remove the private AS number from outbound updates.
+                type: bool
+              next_hop_self:
+                description:
+                  - Enable/disable the next hop calculation for this neighbor.
+                type: bool
+              next_hop_unchanged:
+                description:
+                  - Enable/disable propagation of next hop unchanged for iBGP paths to this neighbor.
+                type: bool
+  operation:
+    description:
+      - Specifies the operation to be performed on the BGP process configured on the device.
+      - Merge will configure the device based on the options specified and negate the configurations that are
+        not specified for that option(i.e, networks, neighbors, etc.) in the task but present in running-configuration.
+      - Replace will remove the existing BGP configuration on the device and re-configure it with the options specified.
+      - Delete will remove the existing BGP configuration from the device.
+    default: merge
+    choices:
+      - merge
+      - replace
+      - delete
+"""
+
+EXAMPLES = """
```

Comment:
Same comments here as from the bgp review - IP addresses for documentation should avoid the 1.1.1.1 etc and instead use:
192.0.2.0/24 
198.51.100.0/24
203.0.113.0/24 

And AS numbers for documentation are in the 64496 - 64511 range. 
Should be applied to these examples and in the sample RETURN block